### DISCUSSION TITLE
Is it possible to connect two nodes without connecting with edges?

### DISCUSSION THREAD

**Discussion by igorDevLima** [antvis/X6#3485] [Q&A]
I would like the nodes to connect just by touching the port, like a Lego. As in the example below:
![Screenshot 2023-04-10 at 09-27-15 React App](https://user-images.githubusercontent.com/116180920/230901463-bf754721-eb3d-4d14-836e-2b158ef34321.png)
![Screenshot 2023-04-10 at 09-28-21 React App](https://user-images.githubusercontent.com/116180920/230901628-247f139f-1d9c-47b4-bf09-83f97ea54068.png)

**Comment by NewByVector** [comment]
It is not currently supported, but we have plans to support this function, but it is not a high priority in our work, so I can't give the schedule. If you are interested, you can mention pr.
